obj2ms3dascii.mjs

```diff
@@ -13,7 +13,7 @@ export function obj2ms3dascii( obj, mtl ) {
   };
 
   let currentObject = null;
-  const objects = [];
+  const unfiltered_objects = [];
```

Comment:
[SHOULD] How is this "unfiltered" against the `object` which will be defined later? I'd prefer add a comment about that.

Reply:
[This](https://github.com/0b5vr/obj2ms3dascii/pull/3#discussion_r810723418) is the substantial issue that takes precedence imo